README.md

```diff
@@ -343,6 +344,57 @@ helm install valheim-server valheim-k8s/valheim-k8s # see repo for full config
 CDK Project for spinning up a Valheim game server on AWS Using ECS Fargate and Amazon EFS is available here:
 [https://github.com/rileydakota/valheim-ecs-fargate-cdk](https://github.com/rileydakota/valheim-ecs-fargate-cdk)
 
+## Deploying to Nomad
+Copy and Paste the following to `valheim.nomad`:
+```
+job "valheim" {
+  datacenters = ["dc1"]
```

Comment:
Will users have to adjust this as well? Or is every nomad default datacenter called "dc1"?

Reply:
"dc1" is the default used in the documentation, but might need to be changed, depending on the cluster it should run on. Should I add a small explanation about the things that a user might want to change? I think its probably alright for someone who uses nomad already, because its a really simple jobfile.